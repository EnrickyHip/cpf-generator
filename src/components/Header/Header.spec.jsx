import { render, screen } from "@testing-library/react"
import { Header } from '.';

describe("<Header />", () => {

  it("should render correctly", () => {
    render(< Header />)
    const h1 = screen.getByRole("heading", { name: 'GERAR / VALIDAR CPF' });

    expect(h1).toBeInTheDocument();
  })

  /* ---------------------------------------- */

  it("should match snapshot", () => {
    const {container} = render(< Header />);
    expect(container.firstChild).toMatchSnapshot();
  })
})