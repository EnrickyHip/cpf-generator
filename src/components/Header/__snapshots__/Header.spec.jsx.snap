// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<header>
  <h1>
    GERAR / VALIDAR CPF
  </h1>
</header>
`;
